src/OmniSharp.Abstractions/Models/v1/FilesChanged/ChangeTypeConverter.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Newtonsoft.Json;
+using System;
+
+namespace OmniSharp.Models.FilesChanged
+{
+    class FileChangeTypeConverter : JsonConverter
+    {
+        public override bool CanConvert(Type objectType) => objectType == typeof(FileChangeType);
+        public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
+        {
+            switch (reader.Value as string)
+            {
+                case "Change":
```

Comment:
These should probably be case-insensitive string matches.

Reply:
Removed this and switched to the automatic serialization offered by json.net, which is case insensitive.

Reply:
Great!